core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2251,6 +2251,15 @@ private SqlDialect nonOrdinalDialect() {
         .ok(expectedBqFormatDatetime);
   }
 
+  @Test void testBigQuerySafeCast() {
```

Comment:
you should also add here a JavaDoc similar to the existing ones for other tests.